src/ConfigParser.cc

```diff
@@ -454,6 +454,32 @@ ConfigParser::NextQuotedOrToEol()
     return token;
 }
 
+bool
+ConfigParser::optionalKvPair(char * &key, char * &value)
```

Comment:
Please remove this and any other PR-unrelated changes from the PR.

Reply:
Rebasing has gotten rid of this